jax/_src/scipy/stats/_core.py

```diff
@@ -148,11 +170,25 @@ def rankdata(
     return .5 * (count[dense] + count[dense - 1] + 1).astype(dtypes.canonicalize_dtype(jnp.float_))
   raise ValueError(f"unknown method '{method}'")
 
-@implements(scipy.stats.sem, lax_description="""\
-Currently the only supported nan_policies are 'propagate' and 'omit'
-""")
+
 @partial(jit, static_argnames=['axis', 'nan_policy', 'keepdims'])
 def sem(a: ArrayLike, axis: int | None = 0, ddof: int = 1, nan_policy: str = "propagate", *, keepdims: bool = False) -> Array:
+  """Compute the standard error of the mean.
+
+  JAX implementation of :func:`scipy.stats.sem`.
+
+  Args:
+    a: arraylike
+    axis: optional integer. If not specified, the input array is flattened.
+    ddof: integer, default=1. The degrees of freedom in the SEM computation.
+    nan_policy: str, default="propagate". JAX supports only "propagate" and
+      "omit".
+    keepdims: bool, default=False. If true, reduced axes are left in the result
+      with size 1.
+
+  Returns:
+    array
```

Comment:
`Array` with link?

Reply:
The return type is already documented via type annotations.

To be honest, I'm not sure what the pydoc recommendation is for listing return values. It's pretty clear from the context that this is an array containing the standard error of the mean of the input, but should we spell that out every time?